robot/packages/cloud_package_manager.go

```diff
@@ -481,7 +468,7 @@ func (m *cloudManager) unpackFile(ctx context.Context, fromFile, toDir string) e
 				return errors.Wrapf(err, "failed to create directory %q", parent)
 			}
 			//nolint:gosec // path sanitized with safeJoin
-			outFile, err := os.OpenFile(path, os.O_RDWR|os.O_CREATE|os.O_TRUNC, info.Mode().Perm())
+			outFile, err := os.OpenFile(path, os.O_RDWR|os.O_CREATE|os.O_TRUNC, 0o600|info.Mode().Perm())
```

Comment:
I think that ml-model files are created with perms `000`. I am surprised that nobody has noticed this yet.